test/e2e/config/docker.yaml

```diff
@@ -69,15 +69,6 @@ providers:
       new: --metrics-addr=:8080
```

Comment:
test/e2e/data/infrastructure-docker/v1.10/bases/cluster-with-topology.yaml should be identical to test/e2e/data/infrastructure-docker/main/bases/cluster-with-topology.yaml